derive_builder/src/options/struct_mode.rs

```diff
@@ -97,6 +124,7 @@ impl From<OptionsBuilder<StructMode>> for (StructOptions, OptionsBuilder<FieldMo
             builder_visibility: m.builder_vis.unwrap_or(m.build_target_vis),
             builder_pattern: b.builder_pattern.unwrap_or_default(),
             build_target_ident: syn::Ident::new(m.build_target_name),
+            derives: m.derive_traits.unwrap_or_default(),
```

Comment:
Depending on which behaviour we want, I'd suggest either
1. unconditionally push `Debug` and `Clone` onto the derives vector
2. only push `Debug` and `Clone` if they are not yet in the vector
3. or default to ```[`Debug`, `Clone`]```, i.e. `unwrap_or(vec![syn::Ident::new("Debug"), syn::Ident::new("Clone")])`